packages/paste-core/components/form-pill-group/src/FormPill.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {CompositeItem} from '@twilio-paste/reakit-library';
+import type {CompositeStateReturn} from '@twilio-paste/reakit-library';
+import {PillCloseIcon} from './PillCloseIcon';
+
+interface FormPillStylesProps {
+  selected?: boolean;
+  element?: string;
+  children: React.ReactNode;
+  onKeyDown?: (event: React.KeyboardEvent<HTMLButtonElement>) => void;
+  onClick?: () => void;
+}
+
+const FormPillStyles = React.forwardRef<HTMLElement, FormPillStylesProps>(
+  ({element = 'FORM_PILL', selected = false, ...props}, ref) => (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      ref={ref}
+      element={element}
+      as="button"
+      type="button"
+      role="option"
+      cursor="pointer"
+      backgroundColor={selected ? 'colorBackgroundPrimaryWeakest' : 'colorBackground'}
+      borderWidth="borderWidth10"
+      borderColor="colorBorderWeak"
+      borderStyle="solid"
+      borderRadius="borderRadius20"
+      paddingX="space30"
+      paddingY="space20"
+      height="30px"
+      display="flex"
+      columnGap="space20"
+      alignItems="center"
+      _hover={{
+        boxShadow: 'shadowBorderPrimaryStronger',
+      }}
+      _focus={{
+        boxShadow: 'shadowBorderPrimaryStronger',
+      }}
+    >
+      {props.children}
+    </Box>
+  )
+);
+
+export interface FormPillProps extends CompositeStateReturn {
+  selected?: boolean;
+  element?: string;
+  children: React.ReactNode;
+  onSelect?: () => void;
```

Comment:
Same concern here, as above on [line 12](https://github.com/twilio-labs/paste/pull/1803/files#diff-f5db32f3b3a094b9a7bd26d634033f83ac1284d421faeebb634fd5f3f90a8502R12). I wonder if we should type these with the event, or if we are not overriding the types from global handlers, maybe type them as such.

Reply:
I typed this in this particular way because I don't actually pass any arguments when this is called. The `events` object that is typically passed (what I believe you're referencing), is not provided.

Reply:
Oh so we are not exposing the events argument? Makes sense then. FWIW I find it a little confusing that we are using the event handler names but they're not actually event handlers, just callbacks. Not blocking, just my two cents.